code/game/machinery/computer/arcade.dm

```diff
@@ -800,6 +807,45 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 			eventdat += "<P ALIGN=Right><a href='byond://?src=[REF(src)];slow=1'>Slow Down</a> <a href='byond://?src=[REF(src)];keepspeed=1'>Continue</a></P>"
 			eventdat += "<P ALIGN=Right><a href='byond://?src=[REF(src)];close=1'>Close</a></P>"
 
+		if(ORION_TRAIL_OLDSHIP)
+			eventdat += "<br>Your crew spots an old ship floating through space. It might have some supplies, but then again it looks rather unsafe."
+			eventdat += "<P ALIGN=Right><a href='byond://?src=[REF(src)];search=1'>Search it</a><a href='byond://?src=[REF(src)];eventclose=1'>Leave it</a></P><P ALIGN=Right><a href='byond://?src=[REF(src)];close=1'>Close</a></P>"
+
+		if(ORION_TRAIL_SEARCH)
+			if(prob(20))
+				var/rescued = add_crewmember()
+				var/oldfood = rand(1,7)
+				var/oldfuel = rand(4,10)
+				food += oldfood
+				fuel += oldfuel
+				eventdat += "<br>As you look through it you find some supplies and a living person!"
+				eventdat += "<br>[rescued] was rescued from the abandoned ship!"
+				eventdat += "<br>You found [oldfood] <b>Food</b> and [oldfuel] <b>Fuel</b>."
+				eventdat += "<P ALIGN=Right><a href='byond://?src=[REF(src)];eventclose=1'>Continue</a></P>"
+				eventdat += "<P ALIGN=Right><a href='byond://?src=[REF(src)];close=1'>Close</a></P>"
+				canContinueEvent = 1
```

Comment:
It looks like these three lines are copied at the end of every outcome. You could just move them down on tab and get rid of 12 line of duplicate code.